yarn.lock

```diff
@@ -98,6 +98,13 @@
     "@phosphor/signaling" "^1.2.2"
     "@phosphor/virtualdom" "^1.1.2"
 
+"@primer/octicons-react@^9.0.0":
```

Comment:
@marcdumais-work Could you please run your tool on new deps to check whether everything is alright? 🙏 

We hope to test and merge today/tomorrow and then release 0.7.2.

Reply:
@akosyakov I did the "deep" [check](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-new-or-updated-npm-production-dependency), recursively covering all production dependencies for the whole repo/PR, and everything looks good.

Reply:
@marcdumais-work thank you